lib/core/aggregator.ex

```diff
@@ -110,7 +110,7 @@ defmodule TelemetryMetricsPrometheus.Core.Aggregator do
     end)
   end
 
-  @spec bucket_measurements(measurements :: [number()], buckets :: Metrics.Distribution.buckets()) ::
+  @spec bucket_measurements(measurements :: [number()], buckets :: tuple() | Keyword.t()) ::
```

Comment:
Is there a better return typespec we could use?

Reply:
We are still just passing the buckets here, so it seems like it should still be `[number(), ...]` as seen here https://github.com/beam-telemetry/telemetry_metrics/pull/72/files#diff-bb595d1269de786b707b36f47a7c46a5L34. For convenience, however, it might make sense to just add the `buckets` type to the `Distribution` module.